docs/Video Streaming for Navigation Apps/Introduction/index.md

```diff
@@ -99,4 +99,36 @@ func hmiLevel(_ oldLevel: SDLHMILevel, didChangeToLevel newLevel: SDLHMILevel) {
 To present a keyboard (such as for searching for navigation destinations), you should use the @![iOS]`SDLScreenManager`!@@![android]`ScreenManager`!@'s keyboard presentation feature. For more information, see the [Popup Menus and Keyboards](Displaying a User Interface/Popup Menus and Keyboards) guide.
 
 ## Navigation Subscription Buttons
-Head units supporting RPC v6.0+ may support navigation-specific subscription buttons for the navigation template. These subscription buttons allow your user to manipulate the map using hard buttons located on car's center console or steering wheel. It is important to support these subscription buttons in order to provide your user with the expected in-car navigation user experience. This is especially true on head units that don't support touch input as there will be no other way for your user to manipulate the map. See [Template Subscription Buttons](Displaying a User Interface/Template Subscription Buttons) for a list of these navigation buttons.
\ No newline at end of file
+Head units supporting RPC v6.0+ may support navigation-specific subscription buttons for the navigation template. These subscription buttons allow your user to manipulate the map using hard buttons located on car's center console or steering wheel. It is important to support these subscription buttons in order to provide your user with the expected in-car navigation user experience. This is especially true on head units that don't support touch input as there will be no other way for your user to manipulate the map. See [Template Subscription Buttons](Displaying a User Interface/Template Subscription Buttons) for a list of these navigation buttons.
+
+## Canceling the Route
+Between your navigation app and the embedded navigation app, only one route should be in progress at a time. When your navigation service is no longer the active service your app will need to cancel its route.
+
+@![iOS]
+!@
```

Comment:
The suggestions are messing up. Please take the raw code of this and insert it here.

@![iOS]
##### Objective-C
```objc
[self.sdlManager.systemCapabilityManager subscribeToCapabilityType:SDLSystemCapabilityTypeAppServices withUpdateHandler:^(SDLSystemCapability * _Nullable capability, BOOL subscribed, NSError * _Nullable error) {
    SDLAppServicesCapabilities *serviceCapabilities = capability.appServicesCapabilities;
    for (SDLAppServiceCapability *serviceCapability in serviceCapabilities.appServices) {
        if ([serviceCapability.updatedAppServiceRecord.serviceManifest.serviceName isEqualToString:<#Your service name#>]) {
            if (!serviceCapability.updatedAppServiceRecord.serviceActive) {
                // Cancel your active route
            }
        }
    }
}];
```

##### Swift
```swift
sdlManager.systemCapabilityManager.subscribe(capabilityType: .appServices) { (systemCapability, subscribed, error) in
    guard let serviceCapabilities = systemCapability?.appServicesCapabilities?.appServices else { return }
    for serviceCapability in serviceCapabilities {
        if serviceCapability.updatedAppServiceRecord.serviceManifest.serviceName == <#Your service name#> {
            if !serviceCapability.updatedAppServiceRecord.serviceActive.boolValue {
                // Cancel your active route
            }
        }
    }
}
```
!@